XamlControlsGallery/ControlPages/TreeViewPage.xaml

```diff
@@ -84,12 +89,12 @@
             </local:ControlExample.Example>
             <local:ControlExample.Xaml>
                 <RichTextBlock>
-                    <Paragraph>&lt;controls:TreeView ItemsSource="{x:Bind DataSource}"/&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;TreeView.ItemTemplate/&gt;</Paragraph>
-                    <Paragraph TextIndent="24">&lt;DataTemplate x:DataType="local:ExplorerItem"/&gt;</Paragraph>
-                    <Paragraph TextIndent="36">&lt;TreeViewItem ItemsSource="{x:Bind Children}" Content="{x:Bind Name}"/&gt;</Paragraph>
+                    <Paragraph>&lt;controls:TreeView ItemsSource="{x:Bind DataSource}"&gt;</Paragraph>
+                    <Paragraph TextIndent="12">&lt;controls:TreeView.ItemTemplate&gt;</Paragraph>
+                    <Paragraph TextIndent="24">&lt;DataTemplate x:DataType="local:ExplorerItem"&gt;</Paragraph>
+                    <Paragraph TextIndent="36">&lt;controls:TreeViewItem ItemsSource="{x:Bind Children}" Content="{x:Bind Name}"/&gt;</Paragraph>
                     <Paragraph TextIndent="24">&lt;DataTemplate/&gt;</Paragraph>
```

Comment:
I think `&lt;DataTemplate/&gt;` doesn't fix the original issue, the closing bracket is still in the wrong place.

Reply:
Whoops, you are totally right. Good catch, thanks.